src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -467,11 +467,11 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
                 SarifOutputVersion = SarifVersion.Current,
                 TargetFileSpecifiers = new[] { inputLogFilePath },
                 OutputFilePath = actualLogFilePath,
-                Quiet = true,
+                Quiet = false,
                 UpdateInputsToCurrentSarif = updateInputsToCurrentSarif,
                 PrettyPrint = true,
                 Optimize = true,
-                Verbose = true // Turn on note-level rules.
+                Level = new List<FailureLevel> { FailureLevel.Error, FailureLevel.Warning, FailureLevel.Note, FailureLevel.None },
```

Comment:
>Level = new List<FailureLevel> { FailureLevel.Error, FailureLevel.Warning, FailureLevel.Note, FailureLevel.None }, [](start = 16, length = 114)

Since Verbose = true enabled all rules. with our current implementation, we have to send all levels.